kernel/src/checkpoint/tests.rs

```diff
@@ -240,10 +318,22 @@ fn test_v1_checkpoint_latest_version_by_default() -> DeltaResult<()> {
     // contain any true values, as the file added is removed in a following commit.
     assert!(data_iter.next().is_none());
 
-    assert_eq!(data_iter.actions_count, 4);
-    assert_eq!(data_iter.add_actions_count, 1);
+    // Finalize and verify checkpoint metadata
+    let size_in_bytes = 10;
+    let metadata = FileMeta {
+        location: Url::parse("memory:///fake_path_2")?,
+        last_modified: 0,
+        size: size_in_bytes,
+    };
+    writer.finalize(&engine, &metadata, data_iter)?;
+    assert_last_checkpoint_contents(
+        &store,
+        2,                                 // version: latest/last version in the log
```

Comment:
weird indent?
```suggestion
        2,  // version: latest/last version in the log
```

Reply:
if `fmt` makes these comments weird/not line up I would say just move to comment above instead of inline (I think inline generally less preferable these days)